### PR TITLE
Fix more beta clippy issues

### PR DESCRIPTION
**Pull request by rbradford**
- **vmm: memory_manager: Use div_ceil()**
- **performance-metrics: Reap child process**
- **test_infra: Silence warnings about zombie spawned processes**

### CODE REVIEW

**Comment by TimePrinciple** [comment]
We can ignore `zombie_process` for now, and see it persists in later releases of beta channel:

https://github.com/rust-lang/rust-clippy/issues/13459

**Comment by rbradford** [comment]
> We can ignore `zombie_process` for now, and see if it persists in later releases of beta channel:
> 
> [rust-lang/rust-clippy#13459](https://github.com/rust-lang/rust-clippy/issues/13459)

I think it's mostly working as intended - I only disabled in for the generic `GuestCommand` use case as I either it's failing to match to a `.wait()` or I couldn't find the missing call in the integration test suite. In the other cases I added calls to `.wait()` where is was missing.

File: performance-metrics/src/performance_tests.rs [hunk 1]
```diff
@@ -518,6 +518,7 @@ pub fn performance_restore_latency(control: &PerformanceTestControl) -> f64 {
         ));
 
         let _ = child.kill();
+        child.wait().unwrap();
```
Comment (TimePrinciple) [inline performance-metrics/src/performance_tests.rs:521]: Should we stick to `.unwrap()`, the same below :)

File: tests/integration.rs [hunk 2]
```diff
@@ -4255,6 +4255,7 @@ mod common_parallel {
         });
 
         let _ = socat_child.kill();
+        let _ = socat_child.wait();
```
Comment (TimePrinciple) [inline tests/integration.rs:4258]: ditto :)

File: tests/integration.rs [hunk 3]
```diff
@@ -6797,7 +6800,8 @@ mod common_parallel {
             );
         });
 
-        cleanup_spdk_nvme();
+        let _ = spdk_child.kill();
+        let _ = spdk_child.wait();
```
Comment (TimePrinciple) [inline tests/integration.rs:6805]: ditto :)

**Review by TimePrinciple — COMMENTED** [review]
Very good work indeed 🧐, I was planning to address this

File: performance-metrics/src/performance_tests.rs (hunk 1 above)
Comment (rbradford) [reply performance-metrics/src/performance_tests.rs:521]: It's kind of all over the place - I tried to adopt to what it was similar to in the same context. There is definitely some scope for cleaning this up in the future.

**Pull request merged**